superset/assets/src/SqlLab/actions/sqlLab.js

```diff
@@ -220,9 +220,9 @@ export function runQuery(query) {
     };
 
     return SupersetClient.post({
-      endpoint: `/superset/sql_json/${window.location.search}`,
-      postPayload,
-      stringify: false,
+      endpoint: '/superset/sql_json/',
```

Comment:
I saw this stringify: false setting is from PR #5896. @williaster is there a special reason to turn off stringify?

Reply:
don't we really want `stringify: false`? I was under the impression that prevented us from sending up `"null"` and `"false"` versus `null` and `false`

And parseMethod determines how we parse the response, not the request

Reply:
you will see we use post method in many, many places, only in sql lab added `stringify: false`. I assume there must be reason to do extra...

Reply:
This is fine as it is, since we're setting `body` instead of `postPayload`, the `stringify` param is no longer used

Reply:
https://github.com/apache/superset/pull/13509

Reply:
https://github.com/apache/superset/issues/13315